Source/CombatExtended/CombatExtended/Projectiles/BulletCE.cs

```diff
@@ -24,7 +29,7 @@ private void LogImpact(Thing hitThing, out LogEntry_DamageResult logEntry)
                     def,
                     null //CoverDef Missing!
                     );
-             if (!(launcher is AmmoThing))
+             if ((launcher is AmmoThing))
```

Comment:
Should remove the double parens as it's not getting negated.  Why is this check even here?

Reply:
I noticed when you commented, I went to change it to 
```csharp
if(launcher is not AmmoThing)
```
But that is a feature of C#9, not what we run. So I moved it back and forgot to add the negation, I will add an == false, which was my intention prior

Reply:
Ok, as for why, it looks like it is intended to prevent cookoff projectiles from registering in the log. As seen in [this commit](https://github.com/CombatExtended-Continued/CombatExtended/commit/fda28f53af5f9f815f43da57edc1e2dd9ab24d1b)